server/test/actions/ApplicantAuthorizationActionTest.java

```diff
@@ -43,7 +43,7 @@ public void when_applicant_and_program_auth_pass_then_moves_to_next_step_in_the_
     ProgramModel activeProgram =
         ProgramBuilder.newActiveProgram()
             .withBlock()
-            .withRequiredQuestion(testQuestionBank().applicantName())
+            .withRequiredQuestion(testQuestionBank().nameApplicantName())
```

Comment:
This ends up being kind of repetitive, but fits the naming convention I'm establishing.